README.md

```diff
@@ -525,6 +528,103 @@ intercom.subscriptions.delete(subscription)
 intercom.subscriptions.all
 ```
 
+#### Articles
+```ruby
+# Create an article
+article = intercom.articles.create(title: "New Article", author_id: "123456")
+
+# Create an article with translations
+article = intercom.articles.create(title: "New Article",
+                                   author_id: "123456",
+                                   translated_content: {fr: {title: "Nouvel Article"}, es: {title: "Articulo nuevo"}})
+
+# Fetch an article
+intercom.articles.find(id: "123456")
+
+# List all articles
+articles = intercom.articles.all
+articles.each { |article| p article.title }
+
+# Update an article
+article.title = "Article Updated!"
+intercom.articles.save(article)
+
+# Update an article's existing translation
+article.translated_content.en.title = "English Updated!"
+intercom.articles.save(article)
+
+# Update an article by adding a new translation
+article.translated_content.fr = {title: "French Added!"}
+intercom.articles.save(article)
+
+# Delete an article
+intercom.articles.delete(article)
+```
+
+#### Collections
+```ruby
+# Create a collection
+collection = intercom.collections.create(name: "New Collection")
+
+# Create a collection with translations
+collection = intercom.collections.create(name: "New Collection",
+                                         translated_content: {fr: {name: "Nouvelle collection"}, es: {name: "Nuevo colección"}})
+
+# Fetch a collection
+intercom.collections.find(id: "123456")
+
+# List all collections
+collections = intercom.collections.all
+collections.each { |collection| p collection.name }
+
+# Update a collection
+collection.name = "Collection updated!"
+intercom.collections.save(collection)
+
+# Update a collection's existing translation
+collection.translated_content.en.name = "English Updated!"
+intercom.collections.save(collection)
+
+# Update a collection by adding a new translation
+collection.translated_content.fr = {name: "French Added", description: "French description"}
```

Comment:
My French is so basic I can't help here, but if we want to improve the example I can give you a Spanish version of this

```suggestion
collection.translated_content.es = {name: "Colección en español", description: "Descripción en español"}
```

name is "Collection in Spanish" and description is "Description in Spanish"